.cargo/config.toml

```diff
@@ -8,3 +8,8 @@ protocol = "sparse"
 # https://github.com/rust-lang/cargo/issues/4133
 [target.aarch64-unknown-linux-gnu]
 linker = "aarch64-linux-gnu-gcc"
+
+[target.wasm32-unknown-unknown]
+rustflags = [
+  "-C", "target-cpu=mvp"
+]
```

Comment:
It means everyone who builds our crates should specify that, otherwise WASM executor will not work.

Reply:
Yep, thanks for changing it. Nice to not have to have this.